### PR TITLE
Add bindings for base/percentile

### LINKED ISSUE #10: base\percentile

### PR DESCRIPTION
**Pull request by alejandrosame**
## Description

This PR adds python bindings for file base/percentile.h 

Fixes #10 

## Type of change

Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Did you run `make test-all`? Yes

## Checklist:

- [x] New Unit tests added
- [x] Unit tests pass locally with my changes

### CODE REVIEW

File: src/bindings/PyDP/base/percentile.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Provides bindings for perentile class
```
Comment (jeamick) [inline src/bindings/PyDP/base/percentile.cpp]: `// Provides bindings for percentile class`

Comment (jeamick) [reply src/bindings/PyDP/base/percentile.cpp]: Typo

**Review by jeamick — DISMISSED** [review]
Sounds ok to me !! ;)

File: src/bindings/PyDP/base/percentile.cpp (hunk 1 above)
Comment (alejandrosame) [reply src/bindings/PyDP/base/percentile.cpp]: Thanks! Fixed on the last commit

**Review by chinmayshah99 — APPROVED** [review]
While we have classes declared as PercentileInt() and PercentileDouble(), we need towards a strategy so that we just call Percentile() and type of datatype is given in function argument.

**Pull request merged**